aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1502,6 +1619,33 @@ static at::Tensor _quantized_convolution_onednn(
     dilation.size() == (decltype(dilation.size()))kSpatialDim,
     func_name, ": dilation should contain ", kSpatialDim, " elements for ",
     kSpatialDim, "D convolution.");
+  bool is_fp8 = weight.scalar_type() == c10::ScalarType::Float8_e4m3fn;
+  if (is_fp8) {
+    TORCH_CHECK(act_dtype == c10::ScalarType::Float8_e4m3fn,
+      func_name, ": expect input tensor to have fp8 data type, but got ", act_dtype);
+    TORCH_CHECK(act_zero_point == 0,
+      func_name, ": fp8 input should not have zero point.");
+    // the current version of oneDNN does not fp8 conv yet
+    // TODO(weiwen) Refine this part when oneDNN supports fp8 conv
+    bool use_ref = true;
+    if (use_ref) {
```

Comment:
remove this `use_ref` as it's the default one and we can add it later when adding the oneDNN support.

Reply:
Sure